Assets/Mirror/Components/NetworkTransformReliable/NetworkTransformReliable.cs

```diff
@@ -33,6 +33,10 @@ public class NetworkTransformReliable : NetworkTransformBase
         [Range(0.00_01f, 1f)]                   // disallow 0 division. 1mm to 1m precision is enough range.
         public float scalePrecision = 0.01f; // 1 cm
 
+        [Header("Snapshot Interpolation")]
+        [Tooltip("Add a small timeline offset to account for decoupled arrival of NetworkTime and NetworkTransform snapshots.\nfixes: https://github.com/MirrorNetworking/Mirror/issues/3427")]
+        public bool timelineOffset = false;
```

Comment:
Needs to be true by default
```suggestion
        public bool timelineOffset = true;
```